### PR TITLE
I2C2MIDI ops

### PR DESCRIPTION
**Pull request by scanner-darkly**
#### What does this PR do?

Adds I2C2MIDI ops.

#### Provide links to any related discussion on [lines](https://llllllll.co/).

https://llllllll.co/t/i2c2midi-a-diy-module-that-translates-i2c-to-midi/40950

#### How should this be manually tested?

See the thread.

#### I have,
* [x] updated `CHANGELOG.md` and `whats_new.md`
* [x] updated the documentation
* [x] updated `help_mode.c` (if applicable)
* [x] run `make format` on each commit
* [x] run tests

### CODE REVIEW

File: docs/ops/i2c2midi.toml [hunk 1]
```diff
@@ -0,0 +1,576 @@
...
+short = "Set minimum note number for MIDI notes to `x` (0..127), using mode `y` (0..3) for channel `ch`"  
+
+["I2M.MAX"]  
+prototype = "I2M.MAX x y"  
+short = "Set maximum note number for MIDI notes to `x` (0..127), using mode `y` (0..3)"  
+description = """
+Set maximum note number for MIDI notes to `x` (0..127), using mode `y` (0..3). Default is `x = 0` and `y = 0`. The following modes are available for notes higher than the maximum: 0) Ignore notes 1) Clamp notes 2) Fold back notes by one octave 3) Fold back notes by multiple octaves. Use `I2M.MAX#` to add channel parameter.
+"""
+
+["I2M.MAX"]
```
Comment (attowatt) [inline docs/ops/i2c2midi.toml:56]:
```
["I2M.MAX#"]
```

File: docs/ops/i2c2midi.toml [hunk 2]
```diff
@@ -0,0 +1,576 @@
...
+["I2M.NRPN.SET"]  
+prototype = "I2M.NRPN.SET x y z"  
+short = "Send MIDI NRPN message for parameter MSB `x` and LSB `y` with value `y` (0..16383), bypassing any slew settings"  
+description = """
+Send MIDI NRPN message for parameter MSB `x` and LSB `y` with value `y` (0..16383), bypassing any slew settings. Use `I2M.NRPN.SET#` to add channel parameter.
+"""
+
+["I2M.NRPN.SET#"]  
+prototype = "I2M.NRPN.SET# ch x y z"  
+short = "Send MIDI NRPN message for parameter MSB `x` and LSB `y` with value `y` (0..16383) on channel `ch`, bypassing any slew settings"
```
Comment (attowatt) [inline docs/ops/i2c2midi.toml:224]: ~~on~~ →  to 
```
short = "Send MIDI NRPN message for parameter MSB `x` and LSB `y` with value `y` (0..16383) to channel `ch`, bypassing any slew settings"
```

File: docs/ops/i2c2midi.toml [hunk 3]
```diff
@@ -0,0 +1,576 @@
...
+["I2M.B.DIR"]    
+prototype = "I2M.B.DIR x"  
+short = "Set the play direction `x` (0..2) of the buffer"
+description = """
+Set the play direction `x` (0..2) of the buffer. The following directions are available: 0) Forward 1) Backward 2) Pingpong. Keep in mind that changing the direction only affects notes that have been already recorded to the buffer before the change in direction; all notes recorded afterwards are recorded relative to the new direction. Default is `x = 0`.
+"""
+
+["I2M.B.SPE"]  
+prototype = "I2M.B.SPE x"  
+short = "Set the playing speed `x` (1..32767) of the buffer. `x = 100` is equivalent to "normal speed", `x = 50` means double the speed, `x = 200` means half the speed, etc."
```
Comment (attowatt) [inline docs/ops/i2c2midi.toml:455]: it seems, the quotation marks (`"`) cause a problem here, and down below...?

**Review by attowatt — COMMENTED** [review]
Added a few little review comments in the docs.

**Comment by scanner-darkly** [comment]
@attowatt thank you, fixed!

**Comment by attowatt** [comment]
Looks perfect now, thanks!

**Pull request merged**